sherpa-onnx/csrc/online-transducer-model.cc

```diff
@@ -92,6 +96,8 @@ std::unique_ptr<OnlineTransducerModel> OnlineTransducerModel::Create(
     const auto &model_type = config.model_type;
     if (model_type == "conformer") {
       return std::make_unique<OnlineConformerTransducerModel>(config);
+    } else if (model_type == "ebranchformer") {
```

Comment:
Can you also update
https://github.com/k2-fsa/sherpa-onnx/blob/41d378a128312acc6da09251dc32891a4922f6ee/sherpa-onnx/csrc/online-transducer-model.cc#L180-L181